packages/frontend/src/components/common/SqlRunner/CatalogTree.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I don't think we need to import React here 🤔

Reply:
using React.FC below

Reply:
Yup, you shouldn't need to import it because our React version knows what to do. 
If you're always referencing `React.X` then no need to import. 
If using named imports, then yes 🤔

Reply:
updated

Reply:
we need `FC` because I use `children` we don't need whole React import because of JSX not requiring it anymore if that's what you mean

Reply:
Yes, it was more on the usage of imports, rather than what you were importing, which I agree 100%(with using `FC` and `children`)